draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -1128,10 +1128,11 @@ https://www.example.org
   </t></li>
   <li><t>
    If a message is received without <x:ref>Transfer-Encoding</x:ref> and with
-   either multiple <x:ref>Content-Length</x:ref> header fields having
-   differing field values or a single Content-Length header field having an
-   invalid value, then the message framing is invalid and
-   the recipient &MUST; treat it as an unrecoverable error.
+   an invalid <x:ref>Content-Length</x:ref> header field, then the message
+   framing is invalid and the recipient &MUST; treat it as an unrecoverable
+   error, unless the field value can be successfully parsed as a
+   comma-separated list (<xref target="abnf.extension"/>), all values in the
+   list are valid, and all values in the list are the same.
```

Comment:
So per https://github.com/web-platform-tests/wpt/pull/10548 `30` and `030` are different values. I guess this technically covers that, but maybe some examples would help?

Reply:
I'm thinking of adding "... when compared numerically."